ivy_tests/test_ivy/helpers/hypothesis_helpers/dtype_helpers.py

```diff
@@ -304,7 +354,7 @@ def get_castable_dtype(draw, available_dtypes, dtype: str, x: Optional[list] = N
     Returns
     -------
     ret
-        A tuple of inputs and castable dtype.
+        A strategy that draws a tuple of inputs and a castable dtype.
```

Comment:
The return seems to be the result of `draw` function, which doesn't return a strategy, I think the previous description is more suitable.